### DISCUSSION TITLE
Welcome to howler.js Discussions!

### DISCUSSION THREAD

**Discussion by goldfire** [goldfire/howler.js#1546] [Announcements]
## 👋 Welcome!
  We're enabling Discussions to provide everyone with a forum to ask questions and get help from the howler.js community without overflowing issues. This space can be used to:

* Ask questions about how to use howler.js
* Ask/discuss anything else related to howler.js or audio on the web
* Ask questions about anything else web/app audio related
* Share anything you've built using howler.js
* Suggest new features/ideas for the library (that don't fit as an issue)
* Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

To get started, comment below with an introduction of yourself and tell us about what you do with howler.js.

**Comment by suterma** [comment]
Hi, my name is [Marcel](https://github.com/suterma). I use howler.js as the audio engine for [Replayer](https://github.com/suterma/replayer-pwa), my little side project. Since [howler.js already has it's own tag on stackoverflow](https://stackoverflow.com/questions/tagged/howler.js), I also suggest to share very specific questions and answers there.

**Comment by suterma** [comment]
I would take the opportunity and suggest some features:

- Make a promise based variant. I have partially implemented this for the Replayer project mentioned above. It feels very easy to work with, when there is a known sequence of sounds
- Implement the loop count feature. Although dismissed with #1158 and #1380 this really feels like a small but useful addition
- Add a logarithmic fade function

**Comment by lustremedia** [comment]
Hi @goldfire 
I just stumbled upon howler.js! Thanks for the GREAT work, this might just fix all my ails I have from jplayer and plyr.io who neglect basic audio needs (e.g. a proper stop() or unloading src etc),  ... and are more or less unmaintained.

Building web radios with livestreams. (Btw two of your lifestream stations on the demo throw basic auth windows and cannot be played)
If you want to exchange these stations with some nice free swiss radio stations let me know (jazz, classic or pop)

**Comment by SumitAvatarUx** [comment]
Hi my name is Sumit, I am facing a sound related issue with IOS devices . 
Issue details:-
Launch the game on iOS device
Play game with enable sound  (make sure audio is audible now)
Lock the device
Unlock the device - audio will not be audible now
Lock the device again
Unlock the device - audio will be audible again

I also check on google but cound not find any fix solution , can any one please help me for this ?
